### PR TITLE
VSCode: add task 'Serial console' and group task with sequence calling

### PR DESCRIPTION
**Pull request by derskythe**
# What's new

- Second try, [first one](https://github.com/flipperdevices/flipperzero-firmware/pull/2081) was close because my parent repo didn't forked from ofw
- Add task "Serial console" to call serial_cli.py
- Add task to show example of group tasks with run application and run console after it
- This example will be useful for those who are not so familiar with VSCode. I've seen this question a lot in Discord

# Verification 

- In VSCode run tasks: "Serial console", "[Debug] Launch App on Flipper with Serial Console"

# Checklist (For Reviewer)

- [x] PR has description of feature/bug or link to Confluence/Jira task
- [x] Description contains actions to verify feature/bugfix
- [x] I've built this code, uploaded it to the device and verified feature/bugfix

### CODE REVIEW

**Comment by skotopes** [comment]
@hedger please take a look when you have time

**Comment by skotopes** [comment]
@derskythe Btw, no need to invite us to collaborate on your repository. When you create PR there is a checkbox, it's enough to keep it in default position so we can merge dev or push minor fixes.

**Comment by derskythe** [comment]
@skotopes ок, sorry, understood.

File: .vscode/example/tasks.json [hunk 1]
```diff
@@ -128,6 +128,33 @@
             "group": "build",
             "type": "shell",
             "command": "./fbt COMPACT=1 DEBUG=0 launch_app APPSRC=${relativeFileDirname}"
+        },
+        {
+            "label": "[Debug] Launch App on Flipper with Serial Console",
+            "dependsOrder": "sequence",
+            "problemMatcher": [],
```
Comment (hedger) [inline .vscode/example/tasks.json]: `            "group": "build",`
is missing

File: .vscode/example/tasks.json [hunk 2]
```diff
@@ -128,6 +128,33 @@
...
+            "dependsOn": [
+                "[Debug] Launch App on Flipper",
+                "Serial Console"
+            ]
+        },
+        {
+            // Press Ctrl+] to quit
+            "label": "Serial Console",
+            "type": "shell",
+            "command": "./scripts/serial_cli.py",
```
Comment (hedger) [inline .vscode/example/tasks.json]: uses system's python that might not have needed dependencies. Change to `./fbt cli`

File: .vscode/example/tasks.json [hunk 3]
```diff
@@ -128,6 +128,33 @@
...
+            "group": "none",
```
Comment (hedger) [inline .vscode/example/tasks.json:146]: for command to be visible in palette, has to be "build"

Also `            "problemMatcher": [],` is requried by [specs](https://code.visualstudio.com/docs/editor/tasks#_compound-tasks)

>Any background/watch tasks used in dependsOn with "dependsOrder": "sequence" must have a problem matcher that tracks when they are "done". The following task runs task Two, task Three, and then task One.

**Review by hedger — CHANGES_REQUESTED** [review]

**Comment by skotopes** [comment]
Please un-draft when ready

**Comment by derskythe** [comment]
Fixed, also added FBT_NO_SYNC=0, because running fbt makes git calls

**Review by hedger — APPROVED** [review]

**Pull request merged**